index.js

```diff
@@ -13,9 +13,11 @@ module.exports = {
     ],
     "rules": Object.assign(
         {
-            'csstree/validator': {ignore: ['background', 'background-color', 'box-shadow']}, // gets upset by sass rgba() function
+            'csstree/validator': {
+                ignore: ['-styled-mixin0', '-styled-mixin1', '-styled-mixin2', '-styled-mixin3', '-styled-mixin4', '-styled-mixin5', '-styled-mixin6'], // these psuedo identifiers are used by styled components for interpolated CSS generated at runtime
```

Comment:
since this is just javascript.... could do something like 

```
ignore: Array.from({length: 10}, (_, i) => `-styled-mixin${i}`),
```

that'll let someone just increase that number if need be, in a sane way. hopefully this rule will get some kind of pattern match feature in the future though